dot/sync/tip_syncer_integration_test.go

```diff
@@ -252,9 +253,10 @@ func TestTipSyncer_handleTick_case3(t *testing.T) {
 	require.Equal(t, block.ToBlockData(), readyBlockData)
 
 	// add pending block w/ full block, but block is not ready as parent is unknown
-	bs := new(syncmocks.BlockState)
-	bs.On("GetHighestFinalisedHeader").Return(fin, nil)
-	bs.On("HasHeader", mock.AnythingOfType("common.Hash")).Return(false, nil)
+	ctrl := gomock.NewController(t)
+	bs := NewMockBlockState(ctrl)
+	bs.EXPECT().GetHighestFinalisedHeader().Return(fin, nil).AnyTimes()
+	bs.EXPECT().HasHeader(gomock.AssignableToTypeOf(common.Hash{})).Return(false, nil).AnyTimes()
```

Comment:
1. Try to find the expected hash value for the argument (just run it against a dummy `common.Hash{1}` and it will log the one to expect)
2. Try to find the number of times it's called.